ui/app/components/clients/running-total.js

```diff
@@ -34,30 +34,24 @@ import { mean } from 'd3-array';
  * @param {array} barChartData - array of objects, object example: { month: '1/22', entity_clients: 11, non_entity_clients: 36, total: 47, namespaces: [] };
  * @param {array} chartLegend - array of objects with key names 'key' and 'label' so data can be stacked
  * @param {object} runningTotals - top level totals from /activity response { clients: 3517, entity_clients: 1593, non_entity_clients: 1924 }
+ * @param {string} timestamp -  ISO timestamp created in serializer to timestamp the response
  *   
  */
 export default class RunningTotal extends Component {
-  get getTotalClients() {
-    return (
-      this.args.chartLegend?.map((legend) => {
-        return {
-          label: legend.label,
-          total: this.args.runningTotals[legend.key],
-        };
-      }) || null
-    );
+  barChartData = this.args.barChartData;
+  runningTotals = this.args.runningTotals;
+
+  get getEntityClientData() {
+    return {
+      runningTotal: this.runningTotals.entity_clients,
+      averageNewClients: Math.round(mean(this.barChartData?.map((d) => d.entity_clients))),
+    };
```

Comment:
This is more of a stylistic comment but using _get_ in a property name makes it sound like it's a function rather than a property. I think `entityClientData` would be more clear on what this represents.